cmd/kyma/alpha/deploy/opts.go

```diff
@@ -1,24 +1,122 @@
 package deploy
 
 import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
 	"time"
 
 	"github.com/kyma-project/cli/internal/cli"
 )
 
+var (
+	defaultDomain         = "local.kyma.dev"
+	defaultVersion        = "latest"
+	kymaProfiles          = []string{"evaluation", "production"}
+	defaultWorkspacePath  = filepath.Join(".", "workspace")
+	defaultResourcePath   = filepath.Join(defaultWorkspacePath, "kyma", "resources")
+	defaultComponentsFile = filepath.Join(defaultWorkspacePath, "kyma", "installation", "resources", "components.yaml")
+)
+
 //Options defines available options for the command
 type Options struct {
 	*cli.Options
-	OverridesYaml  string
-	ComponentsYaml string
-	ResourcesPath  string
-	CancelTimeout  time.Duration
-	QuitTimeout    time.Duration
-	HelmTimeout    time.Duration
-	WorkersCount   int
+	WorkspacePath      string
+	ResourcePath       string
+	OverridesFile      string
+	ComponentsListFile string
+	CancelTimeout      time.Duration
+	QuitTimeout        time.Duration
+	HelmTimeout        time.Duration
+	WorkersCount       int
+	Domain             string
+	TLSCert            string
+	TLSKey             string
+	Version            string
+	Profile            string
 }
 
 //NewOptions creates options with default values
 func NewOptions(o *cli.Options) *Options {
 	return &Options{Options: o}
 }
+
+//GetProfiles return the currently supported profiles
+func (o *Options) getProfiles() []string {
```

Comment:
Usually in Go, getters do not have the prefix "get". This function would be called `profiles()`.

Reply:
Aah, right.. relict from Java times :D Will align the naming to this convenvtion.

Reply:
Fixed .. renamed all getter/is-prefixed methods.

Reply:
"is" prefixed methods are OK, since they are booleans answering a question.